client/menu/menu.js

```diff
@@ -13,7 +13,17 @@
 // limitations under the License.
 
 /**
- * @fileoverview Module for storing question and user data
+ * @fileoverview Adds an importable configuration file for the TIE menu page.
  */
 
-window.tieData = angular.module('tieData', []);
+window.tieMenu = angular.module('tieMenu', ['tieData']);
+
+tieMenu.controller('MenuPageController',
```

Comment:
Oh wait, sorry, rereading over the final comments below, I realize you were saying to pull this into a view (like Learner's View). I'll try and do that in a following PR.